catroid/src/androidTest/java/org/catrobat/catroid/test/utils/RemovingHardCodingStringForObjectSizeUnit.java

```diff
@@ -0,0 +1,120 @@
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.test.utils;
+
+
+import android.test.ActivityInstrumentationTestCase2;
+
+import com.robotium.solo.Solo;
+
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.ui.MainMenuActivity;
+import org.catrobat.catroid.ui.Multilingual;
+import org.catrobat.catroid.ui.MyProjectsActivity;
+
+/**
+ * Created by Null on 3/28/2017.
+ */
+
+public class RemovingHardCodingStringForObjectSizeUnit extends ActivityInstrumentationTestCase2 <MainMenuActivity> {
+
+	public RemovingHardCodingStringForObjectSizeUnit() {
+		super(MainMenuActivity.class);
+	}
+	private Solo solo;
+
+	@Override
+	protected void setUp() throws Exception {
+		solo = new Solo(getInstrumentation(),getActivity());
+	}
+
+	@Override
+	protected void tearDown() throws Exception {
+		solo.finishOpenedActivities();
+	}
+
+	@Override
+	protected void runTest() throws Throwable {
+		super.runTest();
+	}
+	public void testHardCodingStringForKB_MB_GB_B() throws Exception {
+		solo.assertCurrentActivity("MainMenu",MainMenuActivity.class);
+		solo.sendKey(solo.MENU);
+		solo.clickInList(5);
+		solo.clickInList(1);
+		solo.sleep(3000);
+		solo.assertCurrentActivity("Language", Multilingual.class);
+		solo.scrollDown();
+		solo.clickInList(4);
+		solo.sleep(3000);
+		solo.assertCurrentActivity("MainMenu",MainMenuActivity.class);
+		solo.sleep(2000);
+		solo.clickOnButton("Programs");
+		solo.assertCurrentActivity("Programs", MyProjectsActivity.class);
+		solo.sleep(2000);
+		solo.sendKey(solo.MENU);
+		solo.sleep(1000);
+		solo.clickOnMenuItem("Show details");
+		solo.sleep(5000);
+		assertTrue(solo.searchText("KB",1,true));
+		//
+		solo.goBack();
+		solo.assertCurrentActivity("MainMenu",MainMenuActivity.class);
+		solo.sendKey(solo.MENU);
+		solo.clickInList(5);
+		solo.clickInList(1);
+		solo.sleep(3000);
+		solo.assertCurrentActivity("Language", Multilingual.class);
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.clickInList(4);
+		//
+		solo.assertCurrentActivity("MainMenu",MainMenuActivity.class);
+		solo.sleep(2000);
+		solo.clickOnButton("البرامج");
```

Comment:
can you please put all those button names and expected strings into `static final string` variables with english names (e.g. BUTTON_NAME_PROGRAMS) at the top of this test class, so developers who cannot read these languages can understand what this test does